R/converters_SpectronauttoMSstatsFormat.R

```diff
@@ -65,6 +65,8 @@ SpectronauttoMSstatsFormat = function(
         score_filtering = list(pgq = pq_filter, 
                                psm_q = qval_filter),
         columns_to_fill = list("IsotopeLabelType" = "L"))
+    input[, Intensity := ifelse(Intensity == 0, NA, Intensity)]
```

Comment:
Is there any reason we only want to do this with Spectronaut and Skyline but not other converters? i.e. wondering if this could be part of the MSstatsPreprocess function.

Reply:
Ah yes we could do this for others, but I am guessing it's not an issue for them? This would require a closer look at the actual data.

Reply:
Sounds good, fine with staying within Spectronaut and Skyline first and doing a deeper investigation later (maybe as part of the benchmarking work with Anshuman)